crates/sui-core/src/authority.rs

```diff
@@ -214,6 +215,7 @@ pub struct AuthorityMetrics {
     pub(crate) transaction_manager_package_cache_hits: IntCounter,
     pub(crate) transaction_manager_package_cache_misses: IntCounter,
     pub(crate) transaction_manager_package_cache_evictions: IntCounter,
+    pub(crate) transaction_manager_transaction_queue_age: MystenHistogram,
```

Comment:
Maybe add a `_s` suffix to indicate unit.

Reply:
Changed to ms for better precision.